bcda/models/models_test.go

```diff
@@ -833,46 +740,47 @@ func (s *ModelsTestSuite) TestGetMaxBeneCount() {
 	assert.EqualError(err, "invalid request type")
 }
 
-func (s *ModelsTestSuite) TestGetBeneficiaries() {
-	assert := s.Assert()
-	var aco, smallACO, mediumACO, largeACO ACO
-	acoUUID := uuid.Parse(constants.DevACOUUID)
-
-	err := s.db.Find(&aco, "UUID = ?", acoUUID).Error
-	assert.Nil(err)
-	beneficiaries, err := aco.GetBeneficiaries(true)
-	assert.Nil(err)
-	assert.NotNil(beneficiaries)
-	assert.Equal(50, len(beneficiaries))
-
-	// small ACO has 10 benes
-	acoUUID = uuid.Parse(constants.SmallACOUUID)
-	err = s.db.Debug().Find(&smallACO, "UUID = ?", acoUUID).Error
-	assert.Nil(err)
-	beneficiaries, err = smallACO.GetBeneficiaries(true)
-	assert.Nil(err)
-	assert.NotNil(beneficiaries)
-	assert.Equal(10, len(beneficiaries))
-
-	// Medium ACO has 25 benes
-	acoUUID = uuid.Parse(constants.MediumACOUUID)
-	err = s.db.Find(&mediumACO, "UUID = ?", acoUUID).Error
-	assert.Nil(err)
-	beneficiaries, err = mediumACO.GetBeneficiaries(true)
-	assert.Nil(err)
-	assert.NotNil(beneficiaries)
-	assert.Equal(25, len(beneficiaries))
-
-	// Large ACO has 100 benes
-	acoUUID = uuid.Parse(constants.LargeACOUUID)
-	err = s.db.Find(&largeACO, "UUID = ?", acoUUID).Error
-	assert.Nil(err)
-	beneficiaries, err = largeACO.GetBeneficiaries(true)
-	assert.Nil(err)
-	assert.NotNil(beneficiaries)
-	assert.Equal(100, len(beneficiaries))
-
-}
+// TODO: Implement these tests in our service_test.go
```

Comment:
Lots of commented out tests... have they been moved elsehwere?

Reply:
Whoops, forgot to remove these tests. Yes they've all been implemented in service_test.go